app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -256,9 +276,18 @@ def authenticator_view_data
     }.merge(generic_data)
   end
 
+  def piv_cac_view_data
+    {
+      two_factor_authentication_method: two_factor_authentication_method,
+      user_email: current_user.email,
+      remember_device_available: false,
+    }.merge(generic_data)
```

Comment:
Code Climate says this is not tested.

Reply:
I was copying the pattern for the other authentication forms. Found where it was being used. Should be tested now.

Reply:
Same comment about moving this to `piv_cac_verification_controller.rb`.